lessons/topics/01-variables/README.md

```diff
@@ -0,0 +1,20 @@
+# Variables, Scope, Hoisting
+
+In this chapter we will talk about how variables behave in different contexts, how the scope affect the reach of said chunks of data and how we can manage to 
+move the data between blocks and structures
```

Comment:
Why not add this on the previous line?

Reply:
How so?

Reply:
the text "move the data between ..." could be on the previous line, without the line brake.